src/server/CMakeLists.txt

```diff
@@ -69,10 +69,10 @@ cxx_test(multi_test dfly_test_lib LABELS DFLY)
 cxx_test(generic_family_test dfly_test_lib LABELS DFLY)
 cxx_test(hset_family_test dfly_test_lib LABELS DFLY)
 cxx_test(list_family_test dfly_test_lib LABELS DFLY)
-cxx_test(server_family_test dfly_test_lib LABELS DFLY)
 cxx_test(set_family_test dfly_test_lib LABELS DFLY)
 cxx_test(stream_family_test dfly_test_lib LABELS DFLY)
 cxx_test(string_family_test dfly_test_lib LABELS DFLY)
+cxx_test(server_family_test dfly_test_lib LABELS DFLY)
```

Comment:
why this change?

Reply:
looks like issue with rebasing the branch, as at the time this PR was created there was no server_family_test
will fix this